api/v1alpha1/clusterextension_types.go

```diff
@@ -65,6 +65,11 @@ type ClusterExtensionSpec struct {
 	//
 	// Defines the policy for how to handle upgrade constraints
 	UpgradeConstraintPolicy UpgradeConstraintPolicy `json:"upgradeConstraintPolicy,omitempty"`
+
+	//+kubebuilder:Optional
+	//
+	// watchNamespaces indicates which namespaces the operator should watch.
```

Comment:
Since this is on the `ClusterExtension` API I'd change "operator" to "extension":
```suggestion
	// watchNamespaces indicates which namespaces the extension should watch.
```